tests/db_integrity/test_data_updates.py

```diff
@@ -2,112 +2,74 @@
 import pandas as pd
 import numpy as np
 from helpers import get_surface_datapath, get_emissions_datapath, get_footprint_datapath
-from openghg.store import ObsSurface, Emissions
+from openghg.store import ObsSurface
 from openghg.store.base import Datasource
 from openghg.retrieve import get_flux
 from openghg.retrieve import search
-from openghg.standardise import standardise_footprint
+from openghg.standardise import standardise_footprint, standardise_flux, standardise_surface
 from openghg.objectstore import get_bucket
-
-
 from helpers import clear_test_stores
 
 
-def flux_data_read():
-    """
-    Flux data set up.
+@pytest.fixture
+def clear_stores():
```

Comment:
One final detail around this - at this point these fixtures have just ended up as function calls. Re-evaluating these changes in this state, it might make more sense to either:
1. Just add the function calls themselves back to the tests rather than using these fixtures
2. Combine these two fixtures to get and then clear the bucket (could then return bucket value to be used within the tests)

Reply:
All of the tests clear the stores, so they all need to call `clear_stores`, which is why it was autouse, but not all of them need `bucket` as an argument now, because Gareth suggested that I change the standardise functions to only accept 'store' as an argument, and not 'bucket'.

Gareth is also suggesting that `get_bucket()` be replaced by `get_writeable_buckets(name="user")`, so if I add the function calls back then that change becomes harder to make.

Changing the helper functions to take 'bucket' as an argument instead of inheriting through the `auto_bucket' fixture made making the changes Gareth suggested about 'store' instead of 'bucket' harder to do, so I feel like removing some of these fixtures has already made making changes harder. (I admit the global argument in auto_bucket seemed slightly dodgy, although I don't think anything bad could happen because globals are defined at the point where functions are defined, so it only affects code within the same module, which consists of tests and helper functions.)

So, I don't think either option is great, given the changes suggested by Gareth.

Reply:
The first option would be the best one to go with then. There's no reason to have `clear_stores` as a fixture if all it contains is one function call and there's no issue really with having that as a function at the start each test.

Clarity is much more important than brevity when it comes to tests (especially this set of tests). The problem with the autouse was that this reduced clarity especially when the purpose of these specific tests was to very clearly check a particular state of the object store.

Overall this could all be improved by creating a "give me an empty bucket" fixture which creates a new empty bucket and then tidies this up afterwards (which is what I was partially suggesting the fixture at the top could be turned into) but we can use what we have to work with for now.

Reply:
Okay I will put the code back into the tests and remove the fixtures. The combined fixture wouldn't do any harm, but not all of the tests need to use a bucket, so I don't think that aids clarity.

I personally think the autouse fixture for clearing stores is more explicit in this case, since we want every test to start from a blank state; if the code is moved to the tests, then it's not obvious that all of these tests start from a blank slate (in some test modules, clearing stores isn't done uniformly).

Removing the bucket fixture isn't that big of a deal since I think the change to `get_writeable_buckets` should just be search and replace.

Reply:
Thanks for applying that change. I do understand what you're saying - you were aiming to get everything set up from a known base state of an empty object store (and I do think one resolution to consider for future would be to have a `get_empty_bucket` type input).

However, overall we should be careful with what is essentially hidden state and autouse functions and global variables can act as that - I've definitely encountered problems with that kind of setup in the past when trying to debug tests and setting up mocking for instance.